terraform/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 replace (
 	github.com/coreos/go-oidc => github.com/gravitational/go-oidc v0.0.3
+	github.com/gravitational/teleport/api => github.com/gravitational/teleport/api v0.0.0-20210817145431-a8c11241284b
```

Comment:
Curious why do we need this replace clause? The source is the same?

Reply:
It's an artifact :-( Needs to be removed, but I am not quite sure how. Should I update my original branch?